.golangci.yml

```diff
@@ -110,6 +110,26 @@ issues:
       # G404 checks for use of the ordinary non-CPRNG.
       path: private/buf/buflsp/progress.go
       text: "G404:"
+    - linters:
```

Comment:
@doriable can you take a look to see if these exceptions make sense?

Reply:
Yep -- so truncating conversions is definitely a known false positive in the issue I put for the G115 exemptions. I went through each one and they are all good. We already have a clear comment for the one in `report.go` also for converting the version, so this looks good to me.